tests/test_load.py

```diff
@@ -0,0 +1,163 @@
+import pytest
+
+from substratest import assets
+
+# COMPUTE_PLAN_SIZES = [10, 100, 1000, 10000]
+COMPUTE_PLAN_SIZES = [100]
+
+
+@pytest.mark.parametrize('compute_plan_size', COMPUTE_PLAN_SIZES)
+def test_load_single_node_1_branch(compute_plan_size, global_execution_env):
```

Comment:
This is not really clear what's the goal of this test.

The goal of this test is to check that we can create a compute plan with a "lot of tuples".
That's the reason why it is cancelled shortly after creation.

Strictly speaking, I'm not sure it's a load test. This test should not take long to complete.